### ISSUE TITLE
convert  mxnet to pytorch: different inference result from batchnorm layer 

### ISSUE THREAD

**Issue by Jar7** [apache/incubator-mxnet#15717]
Hello, I converted a pretrained model from mxnet to pytorch. And I checked all the argument values, they are the same. I found the different infer results come from batchnorm layer. 
Here is the test code, the output from 2 layers are totally different. Can any one suggest what the problem could be. 
```

import mxnet as mx                                                                                                                  
_,w,w2=mx.model.load_checkpoint('net_mxnet',0)
a=mx.symbol.Variable('data')
bn6_3=mx.sym.BatchNorm(data=a, name='bn6_3', fix_gamma=True)
c=bn6_3.bind(mx.cpu(), args ={'data': mx.nd.ones((1,212)), 'bn6_3_gamma': w['bn6_3_gamma'],'bn6_3_beta':w['bn6_3_beta']},aux_states ={'bn6_3_moving_mean':w2['bn6_3_moving_mean'],'bn6_3_moving_var':w2['bn6_3_moving_var']})
print c.forward()

import torch
from torch import nn

w3=torch.load('net_torch.pth')
inputs = torch.empty(1,212)
inputs = torch.ones_like(inputs)
a=nn.BatchNorm1d(212)
a.eval()
a.state_dict()['weight'].copy_(w3['bn6_3.weight'])
a.state_dict()['bias'].copy_(w3['bn6_3.bias'])
a.state_dict()['running_mean'].copy_(w3['bn6_3.running_mean'])
a.state_dict()['running_var'].copy_(w3['bn6_3.running_var'])
print a(inputs)

assert w3['bn6_3.weight'].numpy().all()==  w['bn6_3_gamma'].asnumpy().all()
assert w3['bn6_3.bias'].numpy().all()==w['bn6_3_beta'].asnumpy().all()
assert w3['bn6_3.running_mean'].numpy().all()==w2['bn6_3_moving_mean'].asnumpy().all()
assert w3['bn6_3.running_var'].numpy().all()==w2['bn6_3_moving_var'].asnumpy().all()
assert a.state_dict()['weight'].numpy().all()==  w['bn6_3_gamma'].asnumpy().all()
assert a.state_dict()['bias'].numpy().all()==w['bn6_3_beta'].asnumpy().all()
assert a.state_dict()['running_mean'].numpy().all()==w2['bn6_3_moving_mean'].asnumpy().all()
assert a.state_dict()['running_var'].numpy().all()==w2['bn6_3_moving_var'].asnumpy().all()`
```

assert operation throws no error.``

**Comment by Jar7** [comment]
fixed. the "eps" in pytorch should be the same as in mxnet.
